src/net/sourceforge/kolmafia/swingui/OptionsFrame.java

```diff
@@ -522,6 +522,10 @@ public ItemOptionsPanel() {
         {"autoSatisfyWithMall", "Buy items from the mall whenever needed"},
         {"autoSatisfyWithCloset", "Take items from the closet whenever needed", "yes"},
         {"autoSatisfyWithStash", "Take items from the clan stash whenever needed"},
+        {
+          "autoSatisfyWithShop",
+          "Take items for sale from the mall shop whenever needed, if an item would be bought"
+        },
```

Comment:
""Take items available in your mall shop whenever needed instead of buying them elsewhere"